sklearn/model_selection/_search.py

```diff
@@ -881,23 +939,41 @@ class GridSearchCV(BaseSearchCV):
             'params'             : [{'kernel': 'poly', 'degree': 2}, ...],
             }
 
-        NOTE that the key ``'params'`` is used to store a list of parameter
+        NOTE
+
+        The key ``'params'`` is used to store a list of parameter
         settings dict for all the parameter candidates.
 
         The ``mean_fit_time``, ``std_fit_time``, ``mean_score_time`` and
         ``std_score_time`` are all in seconds.
 
-    best_estimator_ : estimator
+        For multi-metric evaluation, the scores for all the scorers are
+        available in the ``cv_results_`` dict at the keys ending with that
+        scorer's name (``'_<scorer_name>'``) instead of ``'_score'`` shown
+        above. ('split0_test_precision', 'mean_train_precision' etc.)
+
+    best_estimator_ : estimator or dict
         Estimator that was chosen by the search, i.e. estimator
         which gave highest score (or smallest loss if specified)
         on the left out data. Not available if refit=False.
 
+        For multi-metric evaluation, this attribute is present only if
```

Comment:
I think this wording makes it more confusing for non-multi-metric uses. Just drop the changes for `best_estimator_`. "not available if refit=False" is sufficient for the user to then go and look at `refit` and understand its options.